apps/frontend/core-dapp/src/stores/BalancerStore.ts

```diff
@@ -120,16 +137,28 @@ export class BalancerStore {
    * it to parse user inputs. To estimate the value of a ETH balance without
    * accounting for price impact, use {@link getEthAmountInWstEth}.
    */
-  quoteEthAmountInWstEth(ethAmount: BigNumber): Promise<BigNumber | undefined> {
-    const { wethAddress, wstEthAddress } = this
+  async quoteEthAmountInWstEth(ethAmount: BigNumber): Promise<Quote | undefined> {
+    const { wethAddress, wethDecimals, wstEthAddress } = this
```

Comment:
would it be easier to just read decimalsNumber here? not a big deal tho, just trying to keep the file short and concise

```suggestion
    const { wethAddress, wstEthAddress } = this
    const { decimalsNumber } = this.root.wethStore
```